provisioning/roles/geerlingguy.drupal/tasks/build-composer-project.yml

```diff
@@ -9,15 +25,12 @@
   when: not drupal_site_exists
   failed_when: false
 
-- name: Generate Drupal project with composer package.
+- name: Move Drupal project files to drupal_composer_install_dir
   command: >
-    {{ composer_path }} create-project
-    {{ drupal_composer_project_package }} {{ drupal_composer_install_dir }}
-    {{ drupal_composer_project_options|default('--prefer-dist --no-interaction') }}
-  when: not drupal_site_exists
+    cp -r /tmp/drupal-project/ {{ drupal_composer_install_dir }}/
+    creates={{ drupal_core_path }}/index.php
   become: no
-  environment:
-      COMPOSER_PROCESS_TIMEOUT: 1200
+  when: not drupal_site_exists
```

Comment:
As the destination is a NFS shared directory `mv` would not be faster according to [this](http://rothwerx.com/content/simple-speed-comparison-between-cp-mv-and-rsync). Makes sense also. 

`synchronize` module is just too broken, I can never get it working when it's remote -> remote :/